test/utils/index.js

```diff
@@ -12,11 +15,16 @@ exports.expectSet = (set, subs) => {
   expect(Array.from(set.values())).to.eql(subs)
 }
 
-exports.createNode = (maddr, callback) => {
+const websocketStarMa = '/ip4/127.0.0.1/tcp/14444/ws/p2p-websocket-star/'
```

Comment:
Might be helpful to pull the port from the new `WS_RENDEZVOUS_MULTIADDR` constant that was added, similar to what you're doing in the .aegir.js rendezvous server.